src/background/adblocker.js

```diff
@@ -27,9 +27,73 @@ import asyncSetup from '/utils/setup.js';
 
 import { tabStats, updateTabStats } from './stats.js';
 import { getException } from './exceptions.js';
+import Config, {
+  FLAG_FIREFOX_CONTENT_SCRIPT_SCRIPTLETS,
```

Comment:
it would be easier to review if you would keep the original name

Reply:
It's a waste to make another PR just for changing names... ;)